backend/app.js

```diff
@@ -1,7 +1,123 @@
 const express = require("express");
 const app = express();
 const port = 3000;
+const cors = require("cors");
+app.use(cors());
 
-app.get("/", (req, res) => res.send("Hello World!"));
+app.get("/orders", function(req, res) {
+  // Authorization
+  fs.readFile("credentials.json", (err, content) => {
+    if (err) return console.log("Error loading client secret file:", err);
+    // Authorize a client with credentials, then call the Google Sheets API.
+    authorize(JSON.parse(content), listMajors);
+  });
+
+  /**
+   * Prints the names and majors of students in a sample spreadsheet:
+   * @see https://docs.google.com/spreadsheets/d/1ZNnltFhQluexcZrwWpuaIlMLtDQTeAI2WsoRHWgmELg/edit#gid=0
+   * @param {google.auth.OAuth2} auth The authenticated Google OAuth client.
+   */
+  function listMajors(auth) {
```

Comment:
Can you update the function name and comments in this function to reflect what is actually being done (shouldn't be related to student majors)